data/campaigns/Eastern_Invasion/images/items/yeti-meat-credits.txt

```diff
@@ -0,0 +1 @@
+https://opengameart.org/content/lpc-food
```

Comment:
While the URL is helpful, I think you're meant to copy the "Attribution Notice" from that page too.

That gets complicated, because it refers to CREDITS-food.txt.